osf/models/preprint.py

```diff
@@ -311,7 +311,8 @@ def create_version(cls, create_from_guid, auth):
         last_version = base_guid.versions.order_by('-version').first().version
         data_for_update = {}
         data_for_update['tags'] = source_preprint.tags.all().values_list('name', flat=True)
-        data_for_update['license_type'] = source_preprint.license.node_license
+        if source_preprint.license:
+            data_for_update['license_type'] = source_preprint.license.node_license
```

Comment:
I think we have two types of license, one needs some extra input such as name and year. Please check these info has been copied too.